src/rpc.rs

```diff
@@ -108,6 +109,43 @@ pub struct SystemProperties {
     pub token_symbol: String,
 }
 
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
```

Comment:
Would be good to add a big note to all the types that must align with substrate's.